bridge/client/app/_components/ktb-service-details/ktb-service-details.component.html

```diff
@@ -1,12 +1,12 @@
-<ng-container *ngIf="deployment">
+<ng-container *ngIf="deployment" uitestid="keptn-service-details-deployment">
   <dt-loading-distractor *ngIf="!deployment.sequence, else details">Loading …</dt-loading-distractor>
   <ng-template #details>
     <dt-info-group class="mb-2">
       <dt-info-group-title>
         <div fxFlex fxLayout="row" fxLayoutAlign="flex-start center" fxLayoutGap="15px">
           <div class="mt-1 mb-1" fxLayout="row" fxLayoutAlign="start center" class="service-title">
             <span class="bold" [textContent]="deployment.sequence.getShortImageName() || deployment.service"></span>&nbsp;
-            <dt-tag-list aria-label="deployment-labels">
+            <dt-tag-list aria-label="deployment-labels" uitestid="keptn-service-details-deployment-tags">
```

Comment:
If we want to keep consistent: `keptn-service-details-deploymentTags`

Reply:
Done